CMakeLists.txt

```diff
@@ -1,13 +1,14 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(s2-geometry)
+project(s2geometry VERSION 0.9.0)
```

Comment:
Some of this stuff looks like independent cleanups.

Reply:
The name and version are used in the `setup.py`.